sharing_portal/templates/sharing_portal/includes/stats.html

```diff
@@ -2,18 +2,25 @@
 
 <div class="artifactStats">
   <div class="artifactStats__meta">
-    <span><i class="fa fa-repeat"></i>
+    <span title="launches"><i class="fa fa-rocket"></i>
```

Comment:
We should use the bootstrap `tooltip` field. Here is the example of how we use it for the private icon. This way it'll have a nicer looking popup.

```
<i class="fa fa-eye-slash artifactDetailPrivate" data-toggle="tooltip" data-placement="top" title="This artifact is only visible for those chosen by the author"></i>
```